docker/fullnode/README.md

```diff
@@ -89,9 +99,9 @@ Inspect the database:
 
 ```ls -la suidb/```
 
-## local rpc connectivity issues
+### Investigate local RPC connectivity issues
 
-Update the json-rpc-address in the fullnode config to listen on all addresses:
+Update the `json-rpc-address` in the fullnode config to listen on all addresses:
```

Comment:
When running as a Docker container the container must listen on 0.0.0.0:<port> in order for the 'export' of that port to work to the host.   localhost isn't exported.

Reply:
Is an update needed here then?